policyengine_api/routes/error_routes.py

```diff
@@ -0,0 +1,139 @@
+import json
+from flask import Response, Flask
+from werkzeug.exceptions import (
+    HTTPException,
+    NotFound,
+    BadRequest,
+    Unauthorized,
+    Forbidden,
+    InternalServerError,
+)
+
+
+class ErrorRoutes:
```

Comment:
I wrote this this way purely because I was having difficulty actually registering the handlers properly, but I've found a method within `Blueprint` that is far more similar to what we're already doing, so refactoring against that